core/src/blocktree.rs

```diff
@@ -118,13 +120,108 @@ impl SlotMeta {
     }
 }
 
+#[cfg(feature = "erasure")]
+#[derive(Clone, Debug, Default, Deserialize, Serialize, Eq, PartialEq)]
+/// Erasure coding information
+pub struct ErasureMeta {
+    pub slot: u64,
+    /// Which erasure set in the slot this is.
+    /// Example: Set index `0` covers data indexes `[0, 16)` and coding blobs at `[12,16)`.
+    pub set_index: u64,
+    /// Bitfield representing presence/absence of data blobs
+    pub data: u16,
+    /// Bitfield representing presence/absence of coding blobs
+    pub coding: u8,
```

Comment:
no need to make these so small, we'd hope to vary the slot's erasure config as we move forward

Reply:
Ok I'll make them u64s